docs/advanced_topics/customisation/admin_templates.md

```diff
@@ -75,12 +75,13 @@ To replace the login message, create a template file `dashboard/templates/wagtai
 
 ### `branding_welcome`
 
-To replace the welcome message on the dashboard, create a template file `dashboard/templates/wagtailadmin/home.html` that overrides the block `branding_welcome`:
+```{versionchanged} 4.0
+```
 
-```html+django
-{% extends "wagtailadmin/home.html" %}
+To replace the welcome message on the admin home page (dashboard), create a template file `dashboard/templates/wagtailadmin/home/branding_welcome_title.html` that overrides the content:
```

Comment:
Note that if we went ahead with an approach like #8802, we wouldn’t need to refactor this as much.